src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/CircleSelection.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import type {Node} from 'react';
+import React from 'react';
+import withContainerSize from '../withContainerSize';
+import CircleSelectionRenderer from './CircleSelectionRenderer';
+import type {SelectionData} from './types';
+import circleSelectionStyles from './circleSelection.scss';
+
+type Props = {
+    children?: Node,
+    disabled: boolean,
+    label?: string,
+    maxRadius?: number,
+    minRadius?: number,
+    onChange: (value: ?SelectionData) => void,
+    resizable: boolean,
+    round: boolean,
+    skin: 'filled' | 'outlined',
+    usePercentageValues: boolean,
```

Comment:
This prop should not be needed here anymore, should it?

Reply:
It is needed, because it has to be passed as prop to `withPercentageValues`